3711/3/controllers/places_controller.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../helpers/auth'
+
+class PlacesController < Sinatra::Base
+  include AuthHelper
+
+  register Sinatra::Flash
+
+  configure do
+    set :views, (proc { File.join(root, '../views/places') })
+  end
+
+  get '/places/' do
+    @places = Place.all
+    erb :main
+  end
+
+  get '/places/new' do
+    erb :new
+  end
+
+  get '/places/:id' do
+    @place_id = params['id']
+    erb :index
+  end
+
+  post '/places/new' do
+    Place.create(name: params['name'], note: params['note'], description: params['descr'],
+                 longitude: params['longit'], latitude: params['latit'])
```

Comment:
Please use full size names for params:
`description, longitude or latitude`

Regarding coordinates it's fair enough to call them `lat` and `lng`. It's quite popular naming

Reply:
Consider to create private method `place_params` to filter only necessary field from params with this method
https://apidock.com/rails/Hash/slice
Lines 27-28 will be simplified to `Place.create(place_params)`

Reply:
Use this pattern on params filtering for the rest of controllers please